micrometer-tracing-bridges/micrometer-tracing-bridge-brave/src/test/java/io/micrometer/tracing/brave/bridge/W3CPropagationTest.java

```diff
@@ -299,8 +295,10 @@ void extract_InvalidTracestate_KeyValueDelimiter(W3CPropagationType propagationT
         Map<String, String> invalidHeaders = new HashMap<>();
         invalidHeaders.put(TRACEPARENT, "00-" + TRACE_ID_BASE16 + "-" + SPAN_ID_BASE16 + "-01");
         invalidHeaders.put(TRACESTATE, "foo=bar,test-test");
+        TraceContext actual = propagationType.get().extractor(getter).extract(invalidHeaders).context();
+        TraceContext expected = sampledTraceContext().build();
         assertThat(propagationType.get().extractor(getter).extract(invalidHeaders).context())
-            .isEqualTo(sharedTraceContext().build());
+            .isEqualTo(sampledTraceContext().build());
```

Comment:
It looks like `actual` and `expected` were meant to be plugged into the assertion but aren't.